etc/qgis.profile

```diff
@@ -0,0 +1,46 @@
+# Firejail profile for qgis
+# Description: GIS application
+# This file is overwritten after every install/update
+# Persistent local customizations
+include qgis.local
+# Persistent global definitions
+include globals.local
+
+noblacklist ${HOME}/.config/QtProject.conf
```

Comment:
Add here: `noblacklist ${HOME}/.local/share/QGIS`
Add to `disable-programs.inc`: `blacklist ${HOME}/.local/share/QGIS`